utils/buildcli/buildcli/clang.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
```

Comment:
Haha, I just had to be difficult about this, so I sent up #762. :sweat_smile:

For now, as you suggest I'll copy the header from `utils/build-script`.